### DISCUSSION TITLE
Iron > Refined?

### DISCUSSION THREAD

**Discussion by benhutchison** [erikerlandson/coulomb#596] [General]
@erikerlandson for Scala 3, I feel like Iron is way ahead of Refined. https://github.com/Iltotore/iron

Very well put together. I'd suggest to consider it as a first choice for integration now, where Refined types are required..

**Comment by erikerlandson** [comment]
Sorry, I missed this.  Discussions don't notify my email  :/
I'll take a look

**Comment by erikerlandson** [comment]
The "integration" for `refined` is really just defining things like `AdditiveSemigroup`, `MultiplicativeMonoid`, etc, for `refined` types.  Frankly, if I could persuade @Iltotore to add these to `iron-cats` or `iron-algebra`, there might not even be a need for a `coulomb-iron`.  not that I'd mind doing it, I agree that it seems like a nice package!

**Reply by erikerlandson** [reply]
actually, that's oversimplifying, I'd still probably need the unit and value conversions.  Still, the more fundamental types like `MultiplicativeMonoid[IronType[..]]` would be good candidates for living in `iron`

**Reply by erikerlandson** [reply]
https://github.com/erikerlandson/coulomb/issues/616
